settings.gradle.kts

```diff
@@ -26,6 +26,15 @@ dependencyResolutionManagement {
     }
 }
 
+// Set up aws-crt-kotlin as a composite build
+val localAwsCrtKotlin = file("../aws-crt-kotlin")
+if (localAwsCrtKotlin.exists()) {
+    println("Including aws-crt-kotlin as a composite build")
+    includeBuild(localAwsCrtKotlin)
+} else {
+    println("Could not find aws-crt-kotlin in a sibling directory, not including it")
+}
+
```

Comment:
**Suggestion**: This seems brittle. I'd rather see a more flexible solution [like we have for **aws-sdk-kotlin**](https://github.com/awslabs/aws-sdk-kotlin/blob/26fe2120ae46fa25bc32bce4e5658b472b610db3/settings.gradle.kts#L113-L153), ideally extracted into the repo tools if that's possible.

Reply:
What's brittle about it besides supporting a `local.properties` file? Does anyone even use that configuration option? I didn't see any use of it in our CI / other tooling